templates/spec/controllers/spree/base_controller_spec.rb

```diff
@@ -23,11 +23,14 @@ def index; authorize!(:read, :something); end
       end
 
       context "when http_referrer is present" do
-        before { request.env['HTTP_REFERER'] = '/redirect' }
+        let(:request_referer_path) { '/redirect' }
+        let(:request_referer) { "#{request.protocol}#{request.host}#{request_referer_path}" }
```

Comment:
We've updated the HTTP referers in the specs to match the test host. Rails 7.0 has a new default, wherein it protects against open redirects. Please see https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to for more info.